dialogue-apache-hc5-client/src/test/java/com/palantir/dialogue/hc5/ServerTimingParserTest.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * (c) Copyright 2020 Palantir Technologies Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.dialogue.hc5;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.junit.jupiter.api.Test;
+
+class ServerTimingParserTest {
+
+    @Test
+    void testMissingValue() {
+        assertThat(ServerTimingParser.getServerDurationNanos("missedCache", "missedCache"))
+                .isEqualTo(ServerTimingParser.UNKNOWN);
+    }
+
+    @Test
+    void testSingleMetricWithValue() {
+        assertThat(ServerTimingParser.getServerDurationNanos("cpu;dur=2.4", "cpu"))
```

Comment:
Our servers don't report floating point dur values, but I'd like to include more precise timing at some point.